### ISSUE TITLE
overlapping shows

### ISSUE THREAD

**Issue by identd113** [identd113/hdhr_VCR-AS#4]
Shows can slightly overlap, cause a failed recording on the next show.  Since we send the recording off to curl as a background process, we don't have much control here.  I do not know a clean way to prevent this from ov but wanted to log this for future consideration.

**Comment by identd113** [comment]
If we are within one idle cycle (16 seconds), we could try to delay the idle loop to allow the show to end?

**Comment by identd113** [comment]
I have a added a way to see if there are available tuners.  This is the first step in disallowing overlapping shows.
Most of the work here was in

**Comment by identd113** [comment]
This has been more or less closed with https://github.com/identd113/hdhr_VCR-AS/commit/37e6baa17ad6079b4d3dc6bf1f37c43ff5fb6ef0#diff-5af381b49e80c9f4c6828989c3cc939db4ccd1e8d39ed0142b4f10b9ab1d674b

As a note.  The way we did this is kind of a hack.
We used a pause in the idle() handler, which would halty any process that expects a certain condition, such as time, or status.
The way i see it, the idle queue needs to be consistent at returning on a regular beat, and delaying the queue affects timings on many other things.  
My defense
1. This does not seem to affect the customer Ui at all.
2. It is not very long.  

So, for now , thats the solution.
The correct way of doing this is to set custom return time frames.  Id this was done, we could have large idle loops,which means we would be pretty light usage.
